sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -171,7 +181,6 @@ public virtual bool IsClosed
                 _connection = connection;
                 _retryPolicy = connection.RetryOptions.ToRetryPolicy();
                 ReceiveMode = options.ReceiveMode;
-                PrefetchCount = options.PrefetchCount;
```

Comment:
Maybe if you like the pattern of storing the options and then forward delegating then it would make sense to align this here too.

Reply:
I don't think it is necessary here since none of the properties are publicly mutable.